Gruntfile.js

```diff
@@ -34,154 +34,135 @@ module.exports = function (grunt) {
         babel: {
             all: {
-                options: {
-                    presets: ['es2015']
-                },
+                options: {presets: ['es2015']},
                 files: [{
                     expand: true,
                     cwd: '<%= config.srcScriptsDir %>',
                     src: ['**/*.es.js'],
                     dest: '<%= config.compiledScriptsDir %>',
-                    ext: '.js'
-                }]
-            }
+                    ext: '.js',
+                }],
+            },
         },
         eslint: {
+            options: {fix: grunt.option('fix-eslint')},
             node: {
-                options: {
-                    configFile: '.eslintrc.node'
-                },
-                src: '<%= config.lintFiles.node %>'
+                options: {configFile: '.eslintrc.node'},
+                src: '<%= config.lintFiles.node %>',
             },
             es: {
-                options: {
-                    configFile: '.eslintrc.es'
-                },
-                src: '<%= config.lintFiles.es %>'
+                options: {configFile: '.eslintrc.es'},
+                src: '<%= config.lintFiles.es %>',
             },
             tests: {
-                options: {
-                    configFile: '.eslintrc.es'
-                },
-                src: '<%= config.lintFiles.tests %>'
-            }
-        },
-        jscs: {
-            all: {
-                files: ['<%= config.lintFiles %>']
-            }
+                options: {configFile: '.eslintrc.es'},
+                src: '<%= config.lintFiles.tests %>',
+            },
         },
+        jscs: {all: {files: ['<%= config.lintFiles %>']} },
         swig: {
             all: {
                 expand: true,
                 cwd: '<%= config.templates.sourceDir %>',
                 src: [
-                    '**/*.html'
+                    '**/*.html',
                 ],
-                dest: '<%= config.templates.generatedDir %>'
-            }
+                dest: '<%= config.templates.generatedDir %>',
+            },
         },
         clean: {
             js: [
                 '<%= config.compiledScriptsDir %>',
-                '<%= config.distDir %>'
+                '<%= config.distDir %>',
             ],
             swig: [
-                '<%= config.templates.generatedDir %>/*'
-            ]
+                '<%= config.templates.generatedDir %>/*',
+            ],
         },
-        uglify: {
-            dist: {
-                files: {
-                    '<%= config.distDir %>/language-select.min.js': '<%= config.distDir %>/language-select.js'
-                }
-            }
+        uglify: {dist: {files: {'<%= config.distDir %>/language-select.min.js': '<%= config.distDir %>/language-select.js'} } },
+        ngtemplates: ngTemplatesPaths.generate('', 'app', '<%= config.compiledScriptsDir %>'),
```

Comment:
Considering the tasks you've targeted with no-newline-padding, it seems like you are treating this like code folding, trying to reduce the number of lines in this file so it can be more easily read.

I think other solutions would be better: allow devs to code-fold in their editors; DRY these to a package; etc.